src/localization/en/locale.json

```diff
@@ -0,0 +1,3 @@
+{
+  "welcome":"Welcome! We are really glad to see you!"
```

Comment:
Add more different examples

"OPEN": "{{count:single=Order status was}}{{count:plural=Orders statuses were}} successfully changed to $t(PRODUCTION_LINE.orders.status.OPEN)",